README.md

```diff
@@ -136,6 +136,32 @@ process.on('SIGINT', function () {
 });
 ```
 
+#### Check accessibility
+
+Validates basic level of Gpio compatibility needed by `onoff`
```

Comment:
Please describe the intention of the below program here. I don't think that saying "Validates basic level of Gpio compatibility needed by `onoff`" will mean much to a lot of people.